google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/StreamWriter.java

```diff
@@ -398,7 +397,7 @@ public static StreamWriter.Builder newBuilder(String streamName) {
 
   /** Thread-safe getter of updated TableSchema */
   public synchronized TableSchema getUpdatedSchema() {
-    return singleConnectionOrConnectionPool.getUpdatedSchema();
+    return singleConnectionOrConnectionPool.getUpdatedSchema(this);
```

Comment:
There is a little meaning change in this field now. Previously, it will only return an Updated Schema when a schema update happens during the life time of this StreamWriter. Now it will always return the "current schema" of our knowledge. May worth explain this a bit since Dataflow is going to use this field.

Reply:
Added one line comment